tonic/src/transport/service/discover.rs

```diff
@@ -0,0 +1,50 @@
+use crate::transport::{channel::service::Connection, Endpoint};
+
+use std::{
+    hash::Hash,
+    pin::Pin,
+    task::{Context, Poll},
+};
+use tokio::sync::mpsc::Receiver;
+
+use tokio_stream::Stream;
+use tower::discover::Change;
+
+type DiscoverResult<K, S, E> = Result<Change<K, S>, E>;
```

Comment:
This seems unrelated to the commit message?

Reply:
Fixed.